api/indexer.oas2.json

```diff
@@ -455,9 +651,6 @@
           {
             "$ref": "#/parameters/next"
           },
-          {
-            "$ref": "#/parameters/round"
```

Comment:
Will this impact sdk backwards compatibility? Is this removed for both algod and indexer? Please comment.

Reply:
Yes, anyone who was providing this argument with an SDK will have a compilation error, and anyone who was using without the SDK will receive a 400 (since the backend will not know what "round" is for). After talking with @winder and @algoanne I believe this is preferable to the current behavior — where the round parameter is being silently ignored, the latest round's data is returned, but the caller thinks they are getting a historical value from an old round.